eox_hooks/edxapp_wrapper/backends/models.py

```diff
@@ -1,13 +1,14 @@
 """
 Simple backend that returns the platform's models
 """
+# pylint: disable=import-error,import-outside-toplevel
```

Comment:
this disable everything and that is a thing that we don't want to do

Reply:
In a backend? Where we're going to have a lot of import errors and import-outside-top-level?

Reply:
it concerns me that we won't catch new errors